### ISSUE TITLE
Make crearer, that you _can_ theoretically train your own LLM models

### ISSUE THREAD

**Issue by robinmanuelthiel** [Azure/intro-to-intelligent-apps#5]
The text says, I **can't bring** my own data into an LLM because the models are pre-trained, so the only way to get more information in is to retrain the model, which is an expensive and time consuming process.

https://github.com/Azure/intro-to-intelligent-apps/blob/cf155fca1301b0b730b1c754575cd01996bf7eea/labs/03-orchestration/02-Embeddings/embeddings.ipynb#L82

So this statement is contradictory. I _can_ do it, it's just very expensive and resource intensive and time consuming. I suggest to adjust that to avoid confusion.

**Comment by MarkWme** [comment]
Yes, I get that the statement is a little confusing. Just trying to convey that you can't add your own data to the existing models that the AI is using, but you can use techniques to achieve something similar, which is what embeddings are for. Will look at rewording this. Thanks!
